R/type-tibble.R

```diff
@@ -1,5 +1,6 @@
 # Coercion ----------------------------------------------------------------
 
+#' @include type2.R
```

Comment:
Why is the `@include` needed?

Reply:
It's necessary, because otherwise `type-tibble.R` will be processed before `type2.R` . Could achieve the same with renaming, but this is brittler.